man/man8/zgenhostid.8

```diff
@@ -0,0 +1,38 @@
+.TH zgenhostid 8
+.SH NAME
+zgenhostid \- generate and store a hostid in /etc/hostid
+.SH SYNOPSIS
+.LP
+.nf
+\fBzgenhostid\fR [hostid]
+.fi
+.SH DESCRIPTION
+If \fB/etc/hostid\fR does not exist, create it and store a hostid in
+it.  If the user provides \fBhostid\fR on the command line, store that value.
+Otherwise, randomly generate a value to store.
+
+This emulates the \fBgenhostid(1)\fR utility, and is provided for users on
+SUSE which does not include the utility.
```

Comment:
Would it be worth not mention SUSE directly? The reason I ask, do all other distributions have a `genhostid`?